README.md

```diff
@@ -89,10 +89,6 @@ The latest container image can be found at:
 * `quay.io/coreos/kube-state-metrics:v1.9.7`
 * `k8s.gcr.io/kube-state-metrics:v1.9.7`
```

Comment:
FYI, I found new image pipeline change image path forMetrics Server, now it is published on `k8s.gcr.io/metrics-server/metrics-server:v0.3.7`, I would expect that new kube-state-metrics image will be available on `k8s.gcr.io/kube-state-metrics/kube-state-metrics`